packages/core/src/payment/payment.ts

```diff
@@ -244,4 +244,5 @@ export interface StripeV3FormattedPayload {
     bigpay_token?: {
         token: string;
     };
+    channel_name?: string;
```

Comment:
Does this need to be optional?

Reply:
This implementation is for stripe upe, until we make it for stripe V3 we can remove the optional because they share the interface